wiki/jee.md

```diff
@@ -361,6 +363,102 @@ In order to do this, you must provide the following content in the
 
 ... here, the `<class-loader delegate="false" />` part is the relevant bit.
 
+### Deactivate MoxyJson
```

Comment:
I don't think that should be a problem. However I have to check back with our legal department as well.